electron/dev_tools/rebuild-native.js

```diff
@@ -0,0 +1,139 @@
+/**
```

Comment:
Dev-only script to rebuild native modules for Electron consumption. Can mostly be ignored.